index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Portfolio _one</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" crossorigin />
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <header>
+        <nav class="buttons_nav">
+            <h2 class="logo"><a href="#">My Logo</a></h2>
```

Comment:
- Hey, the logo shouldn't be wrapped in the nav, as it is not a nav item. You should move the logo out of the nav tag but it should still remain in the header tag.
- Also, the logo is fine wrapped in the `a` tag, kindly remove the `h2` you used as a wrapper to improve your semantic usage.